package.json

```diff
@@ -10,15 +10,15 @@
     "url": "git@github.com:resin-io/node-resin-lint"
   },
   "scripts": {
-    "build": "npm run prettify && tsc && npm run lint",
-    "lint": "./bin/resin-lint lib/",
+    "build": "tsc && npm run prettify && npm run lint",
+    "lint": "node ./bin/resin-lint lib/ && node ./bin/resin-lint --typescript lib/",
     "prepublish": "require-npm4-to-publish",
     "prepublishOnly": "npm run test",
-    "prettify": "prettier --config ./config/.prettierrc --write \"lib/**/*.ts\"",
+    "prettify": "node ./bin/resin-lint --typescript --fix lib/",
```

Comment:
Switched to using the built-in auto fixing, it essentially acts as a test in itself